app/controllers/events/past.js

```diff
@@ -0,0 +1,69 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'name',
+      template     : 'components/ui-table/cell/cell-event',
+      title        : 'Name'
+    },
+    {
+      propertyName : 'startTime',
+      template     : 'components/ui-table/cell/cell-date',
+      title        : 'Date'
+    },
+    {
+      propertyName : 'roles',
+      template     : 'components/ui-table/cell/cell-roles',
+      title        : 'Roles'
+    },
+    {
+      propertyName : 'sessions',
+      template     : 'components/ui-table/cell/cell-sessions',
+      title        : 'Sessions'
+    },
+    {
+      propertyName : 'speakers',
+      title        : 'Speakers'
+    },
+    {
+      propertyName : 'tickets',
+      template     : 'components/ui-table/cell/cell-tickets',
+      title        : 'Tickets'
+    },
+    {
+      propertyName : 'url',
+      title        : 'Public URL',
+      class        : 'hide'
+    },
+    {
+      template : 'components/ui-table/cell/cell-buttons',
+      title    : 'Actions'
+    }
+  ],
+  data: [{
```

Comment:
@anu-007 We have #155 for moving dummy data from controllers to routes that was introduced in early commits not for new additions. Please add this to the respective route file.